docs/extensions.adoc

```diff
@@ -49,6 +49,10 @@ Instead, the quality should be based on the extension's own merit.
 |====
 |Name |Runtime |Description
 
+|https://github.com/uniqueck/asciidoctorj-office-extension[Asciidoctor Office]
+|AsciidoctorJ
+|Provides features to convert excel sheets to asciidoc tables and powerpoint slides to images.
```

Comment:
Can you write "asciidoc" as "AsciiDoc", excel as "Microsoft Excel" and powerpoint as "Microsoft PowerPoint"? Thanks!

Reply:
Ok is done.
For the missing microsoft part, my idea was to also support Open Document Format, but that is not so easy. So I think for the first time the Microsoft Format is ok.

Reply:
I think that's a good goal. The reason I requested adding "Microsoft" is because that's the proper name, not just "Excel" (so it's legally required to write it that way).